### PR TITLE
修复开启屏幕分享时会导致其她用户白屏的问题

### PR DESCRIPTION
**Pull request by TNTcraftHIM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,45 @@ Secondary development based on [edumeet](https://github.com/edumeet/edumeet).
 
 ## start with docker
 
-To view the full documentation, please visit: [Deployment Video Meeting](https://tailchat.msgbyte.com/docs/meeting/deployment)
+#### Clone Repo
+```bash
+git clone https://github.com/TNTcraftHIM/tailchat-meeting --depth=1
```
Comment (moonrailgun) [inline README.md]: ?

Comment (TNTcraftHIM) [reply README.md]: > ?

为了方便自己部署在不同服务器上测试改了一下readme，因为发现好像link里的教程没提到docker-compose build的部分，但是readme里有提到，以为大佬您已经不关注这个项目了，非常抱歉，刚刚已经amend了

Comment (moonrailgun) [reply README.md]: can you remove this section? i dont think its a good idea to keep clone repo from other repo if you wanna merge this pr into this repo

Comment (moonrailgun) [reply README.md]: thanks

**Review by moonrailgun — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TNTcraftHIM) [reply README.md]: > can you remove this section? i dont think its a good idea to keep clone repo from other repo if you wanna merge this pr into this repo

Sure, done

**Pull request merged**